ansible/roles/setup_desktop/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+# Desktop environment specific packages and configuration
+- name: Add ucaresystem-core repository
+  ansible.builtin.apt_repository:
+    repo: "ppa:utappia/stable"
+    state: present
```

Comment:
This is related to "utility" section, repo should be declared in group_vars, not hardcoded